allennlp/common/registrable.py

```diff
@@ -38,11 +41,14 @@ class Registrable(FromParams):
     a subclass to load all other subclasses and the abstract class).
     """
 
-    _registry: Dict[Type, Dict[str, Tuple[Type, Optional[str]]]] = defaultdict(dict)
+    _registry: ClassVar[DefaultDict[type, _SubclassRegistry]] = defaultdict(dict)
```

Comment:
Should this be:

```suggestion
    _registry: ClassVar[DefaultDict[Type, _SubclassRegistry]] = defaultdict(dict)
```

Reply:
Same deal as above. We can't make too many assumptions about whatever the key for the default dictionary. All we know is that it's going to be a type.

I think... 🤔